reference/settings.v3.yml

```
@@ -30,146 +30,194 @@ servers:
   - description: Production API Gateway
     url: 'https://api.bigcommerce.com'
 paths:
-  '/stores/{store_hash}/v3/settings/analytics':
+  '/stores/{store_hash}/v3/settings/data-solutions/web-analytics':
     get:
-      summary: Get All Web Analytics Providers
-      description: Returns a list of web analytics providers.
+      operationId: get-web-analytics
+      tags:
+        - Web Analytics
+      summary: Get All Web Analytics
+      description: 'Returns a list of all available Web Analytics data for a specified channel. If `channel_id` is not set, the endpoint returns the analytics for the default channel.'
       parameters:
-        - $ref: '#/components/parameters/ChannelIdParam'
+        - name: store_hash
+          in: path
+          required: true
+          description: Unique identifier for a store.
+          schema:
+            type: string
+        - name: channel_id
+          in: query
+          description: 'ID of the storefront channel. Defaults to the ID of the default (global) channel.'
+          schema:
+            type: integer
       responses:
         '200':
-          description: OK
+          description: |-
+            Successful operation. 
+
+            <!-- theme: info -->
+            > #### Note:
+            > - The `200` response body returns an array of all analytics objects. View each individual analytic by clicking the dropdown next to **data**.
+            > - For Google Analytic, the response includes `tracking_code` or `property_id` depending on the `version`.
           content:
             application/json:
               schema:
-                type: object
-                properties:
-                  data:
-                    $ref: '#/components/schemas/AnalyticsProviders'
-                  meta:
-                    type: object
-      tags:
-        - Analytics
-    parameters:
-      - $ref: '#/components/parameters/store_hash'
-  '/stores/{store_hash}/v3/settings/analytics/{id}':
+                $ref: '#/components/schemas/Analytics'
+        '400':
+          description: Bad request. Authentication Required.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse400'
+  '/stores/{store_hash}/v3/settings/data-solutions/web-analytics/{id}':
     get:
-      summary: Get a Web Analytics Provider
-      description: Returns a single web analytics provider data for a default channel.
+      operationId: get-a-web-analytic
+      tags:
+        - Web Analytics
+      summary: Get a Web Analytic
+      description: |-
+        Returns a single Web Analytic for a specified channel. 
```

```suggestion
        Returns a single web analytic for a specified channel. 
```
I know we aren't consistent across specs, but it feels like this doesn't need to be capitalized. If it does, we would want to be consistent within the same spec file like on line 92 which reads "**Note**: Web analytic ID 5 is no longer in use. "